packages/framework/assets/js/common/validation/customizeFpValidator.js

```diff
@@ -33,39 +45,50 @@ FpJsFormValidator.customizeMethods._submitForm = FpJsFormValidator.customizeMeth
 //   there can be loaded callbacks from last form submit which can cause duplicated form error windows
 // (the rest is copy&pasted from original method; eg. ajax validation)
 FpJsFormValidator.customizeMethods.submitForm = function (event) {
+
+    if ($(':focus').hasClass('js-no-validate-button')) {
+        return;
+    }
+
```

Comment:
is the comment above (starting with `// Custom behavior:`) still up to date? 
Shouldn't be something from the new behavior mentioned there?

Reply:
You are right, I added new comment.